src/adagrams.js

```diff
@@ -1,15 +1,169 @@
+/**
+ *
+ * creates an array of 10 random letters selected from LETTER_POOL
+ * @return {array of 10 random letters} output
+ */
```

Comment:
Love the us of JSDoc!

In the curly braces, we want the type of the data coming back, not a description of the data. The description comes after the braces. So it would be:
```
@return {String[]} array of 10 random letters
```